CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+## neptune-lightgbm 1.0.0
+
+ ### Changes
+ - `NeptuneCallback` now accepts a namespace `Handler` as an alternative to `Run` for the `run` argument. This means that
+   you can call it like `NeptuneCallback(run=run["some/namespace/"])` to log everything to the `some/namespace/`
+   location of the run.
+
+ ### Breaking changes
+ - We removed support for the legacy Neptune client API. Use it with `import neptune.legacy as neptune`.
```

Comment:
As @normandy7 noticed, this line is not needed if not misleading.

Reply:
removed. Thanks!